business/istio_config.go

```diff
@@ -795,9 +795,12 @@ func (in *IstioConfigService) UpdateIstioConfigDetail(ctx context.Context, clust
 	kubeCache.Refresh(namespace)
 
 	// Re-run validations for that object to refresh the validation cache.
-	if _, _, err := in.businessLayer.Validations.ValidateIstioObject(ctx, cluster, namespace, resourceType, name); err != nil {
-		// Logging the error and swallowing it since the object was updated successfully.
-		log.Errorf("Error while validating Istio object: %s", err)
+	c := config.Get()
+	if c.IsValidationsEnabled() {
```

Comment:
Instead of using the global var `config.Get()`
```suggestion
	if in.conf.IsValidationsEnabled() {
```

Reply:
Is it simple enough to add a unit test for this? That validations aren't performed on update if disabled in the config?

Reply:
Hi @nrfox sorry for the delay, I was on PTO.

Yeah, validations are disabled also when updating, as you can see in the following image, there is an error on the Gateway name and there is no validation:

![image](https://github.com/user-attachments/assets/d96baf15-c3db-42cd-94ec-be91928bfa63)

Also, in the List, there is the N/A indication:

![image](https://github.com/user-attachments/assets/90b8bd0f-0fc9-41b3-9b42-d2ab45cf9221)

I will review your comments. I was thinking on the testing side, I need to check if a test is doable, I guess so.